samples/ADC/Program.cs

```diff
@@ -14,37 +13,37 @@ public class Program
     {
         public static void Main()
         {
-            string devs = AdcController.GetDeviceSelector();
+            // nanoFramework provides a single "virtual" ADC controller, even if the target has multiple ones.
+            AdcController adc = AdcController.GetDefault();
 
-            Debug.WriteLine("devs=" + devs);
+            // Minimum value is usually 0, but maximum depends on the board and the actual physical analog-to-digital converter.
+            // For example, STM32F769 has 12bit ADC, so the maximum value is 4095.
+            int maximum = adc.MaxValue;
+            int minimum = adc.MinValue;
+            Debug.WriteLine($"ADC minimum value is {minimum}, maximum - {maximum}");
 
-            AdcController adc1 = AdcController.GetDefault();
+            // Analog channels in nanoFramework are defined sequentially: 0, 1, 2, etc. However, these number are mapped inside firmware,
+            // and it is not obvious of how manufacturers datasheet corresponds to the nanoFramework channel numbers.
+            // It is different for every target board. For example, Channel 0 for STM32F769DISCOVERY board corresponds to ADC1_IN6,
+            // which is mapped to PA6 MCU pin, which is then routed to A0 pin on Arduino headers.
+            // Try looking for such information in reference target documentation pages: https://docs.nanoframework.net/content/reference-targets/index.html
 
-            int max1 = adc1.MaxValue;
-            int min1 = adc1.MinValue;
+            // For this sample, let's just open Channel 0. There's a good chance it is present on most boards.
```

Comment:
Let's just say that we're opening channel 0, OK? No need to add the judgmental second part. If 0 is not available in any given board, pretty sure the developer working on it will be able to take corrective steps.